Automattic/vip
# chore(deps): replace the deprecated `opn` with `open`

## Description

This PR replaces the deprecated `opn` package with `open`.

Ref: https://www.npmjs.com/package/opn

## Steps to Test

1. Check out PR.
2. Run `npm run build`
3. Run `./dist/bin/vip.js login`
4. Observe that the browser opens.
